src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -292,6 +294,10 @@ object ConductrPlugin extends AutoPlugin {
         conductrImageVersion.toSeq ++
         conductrImage.withFlag(Flags.image) ++
         nrOfContainers.withFlag(Flags.nrOfContainers) ++
+        nrOfInstances.map {
```

Comment:
The arg reflects the arity of the arg in the CLI itself, so I think we're good here.